Unable to build on Manjaro

**Describe the bug**
I'm unable to build this tool from source

**To Reproduce**
Steps to reproduce the behavior:
1. Run 'yay -Sy linux-wifi-hotspot'
2. Install all the required dependencies
3. Wait for make to start
4. See error

**Expected behavior**
The program compiles fine

**Desktop (please complete the following information):**
 - OS: Manjaro
 - Version 20.1.1

**Terminal output**
```
==> Starting build()...
mkdir -p build
Run 'sudo make install' for installation.
Run 'sudo make uninstall' for uninstallation.
cd src && make
make[1]: Entering directory '/home/ag/.cache/yay/linux-wifi-hotspot/src/linux-wifi-hotspot-3.0.0/src'
/usr/bin/glib-compile-resources ui/glade/wifih.gresource.xml --target=ui/resources.c --sourcedir=ui/glade --generate-source
gcc -c -o ../build/main.o ui/main.c `pkg-config --cflags gtk+-3.0`
gcc -c -o ../build/ui.o ui/ui.c `pkg-config --cflags gtk+-3.0`
gcc -c -o ../build/h_prop.o ui/h_prop.c `pkg-config --cflags gtk+-3.0`
gcc -c -o ../build/util.o ui/util.c `pkg-config --cflags gtk+-3.0`
g++ -c -o ../build/read_config.o ui/read_config.cpp
make[1]: *** No rule to make target '../build/resources.o', needed by '../build/wihotspot-gui'.  Stop.
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/home/ag/.cache/yay/linux-wifi-hotspot/src/linux-wifi-hotspot-3.0.0/src'
make: *** [Makefile:5: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...
error making: linux-wifi-hotspot
```

--- a/src/Makefile
+++ b/src/Makefile
@@ -20,19 +20,26 @@ BUILT_SRC = resources.c
 _OBJ = main.o ui.o h_prop.o util.o read_config.o $(BUILT_SRC:.c=.o)
 OBJ = $(patsubst %,$(ODIR)/%,$(_OBJ))
 
-all: resources.c $(ODIR)/wihotspot-gui
+# Determine this makefile's path.
+# Be sure to place this BEFORE `include` directives, if any.
+THIS_FILE := $(lastword $(MAKEFILE_LIST))
+
+.PHONY: clean
+
+all: resources.c
 
 resources.c: ui/glade/wifih.gresource.xml ui/glade/wifih.ui
 	$(GLIB_COMPILE_RESOURCES) ui/glade/wifih.gresource.xml --target=ui/$@ --sourcedir=ui/glade --generate-source
-
+	@$(MAKE) -f $(THIS_FILE) wihotspot-gui
+	
 $(ODIR)/%.o: ui/%.c
 	$(CC) -c -o $@ $< $(CFLAGS)
 
 $(ODIR)/%.o: ui/%.cpp
 	g++ -c -o $@ $<
 
-$(ODIR)/wihotspot-gui: $(OBJ)
-	$(CC) -o $@ $^ $(CFLAGS) $(LIBS)
+wihotspot-gui: $(OBJ)
+	$(CC) -o $(ODIR)/$@ $^ $(CFLAGS) $(LIBS)
 
 install: $(ODIR)/wihotspot-gui
 	mkdir -p $(DESTDIR)$(APP_DIR)
@@ -53,8 +60,6 @@ clean-old:
 	rm -rf $(DESTDIR)/usr/share/wihotspot
 	rm -f $(DESTDIR)/usr/bin/wihotspot_gui
 	rm -f $(DESTDIR)/usr/bin/wihotspot
-
-.PHONY: clean
 
 clean:
 	rm -f $(ODIR)/*.o

